### ISSUE TITLE
(#100) Tried accessing nonexisting field

### ISSUE THREAD

**Issue by PaulieScanlon** [kennetpostigo/react-facebook-login-component#9]
Hi, bit of a React noob here, i've implemented the login component as per the usage example and it does authorise my app but i get an error in the console if i log out the response.

Any ideas?

(#100) Tried accessing nonexisting field (undefined) on node type (User)

**Comment by kennetpostigo** [comment]
@PaulieScanlon This seems to be a duplicate of #6 which was solved by adding fields props:
```js
<FacebookLogin socialId="yourAppID"
                       language="en_US"
                       scope="public_profile,email"
                       fields="id,name,email"
                       responseHandler={this.responseFacebook}
                       xfbml={true}
                       version="v2.5"
                       class="facebook-login"
                       buttonText="Login With Facebook"/>
```

Can you try this and lmk if it works as it worked for another user.

**Comment by PaulieScanlon** [comment]
problem solved, i found the answer on [#6](https://github.com/kennetpostigo/react-facebook-login-component/issues/6)

**update, apologies, cross over on posts there! thanks @kennetpostigo for your comment anyway!
